src/main/java/jenkins/plugins/logstash/kibana/EmbeddedLogActionFactory.java

```diff
@@ -28,8 +29,8 @@
 
     @Override
     public Collection<? extends Action> createFor(Run target) {
-        return Arrays.<Action>asList(new KibanaEmbeddedLogAction(target), 
-                new ElasticsearchIncrementalLogAction(target));
+        return target instanceof WorkflowRun ? Collections.singleton(new KibanaEmbeddedLogAction(target)) :
```

Comment:
`ElasticsearchIncrementalLogAction` is useless for `WorkflowRun` since the regular console link works.